reviews/sandbox/2022/2022-Confidential-Containers-annual.md

```diff
@@ -0,0 +1,155 @@
+# Confidential Containers 2022 Annual Review
+
+## Background
+
+Confidential Containers is a CNCF Sandbox project that enables cloud native
+confidential computing by taking advantage of a variety of hardware platforms
+and technologies. The project was accepted as a CNCF Sandbox in March 2022.
+
+Confidential Containers builds on existing and emerging hardware security
+technologies such as Intel SGX, Intel TDX, AMD SEV and IBM Z Secure Execution,
```

Comment:
```suggestion
technologies such as Intel SGX, Intel TDX, AMD SEV, and IBM Z Secure Execution,
```
I am an Oxford comma kind of guy. Without the comma it reads that AMDSEV and IBM Z Secure Execution requires/expects the use of both technologies.

Reply:
Fixed, thanks.